src/lib/kit/components/Views/NumberWithScaleView/NumberWithScaleView.tsx

```diff
@@ -10,14 +10,16 @@ import './NumberWithScaleView.scss';
 
 const b = block('number-with-scale-view');
 
+const POPOVER_PLACEMENT: PopoverProps['placement'] = ['bottom', 'top'];
```

Comment:
these changes do not apply to the current task, you should remove them, if you want to change imports and parameters, it is better to do it in a separate task